### PR TITLE
MAISTRA-793 removed support for ior

### PR DESCRIPTION
**Pull request by dmitri-d**

### CODE REVIEW

File: manifests-maistra/1.0.0/maistraoperator.v1.0.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -124,6 +123,9 @@ spec:
...
...gytbM4uxyNru5FxkY/uAxIAo1jb3/ZuraX/JbBDLnIqz2JRqzutUDLLWatEVWfSn/57/11MNYWUgtdW+s3cYXNl6t/zfjKJruxNzke3dQ9KjYbG5czg1F52ai/76l/Sy1utU3DaTJArOo084xSApJtluNWg0mlqftdxlWQiuk5j6NHl9331PeoY06iuL596t7e7HSQ9CNUkUykrNh/HkYniD9CzHUfqJNTkXIT0CAw4OE7Pv1khPkR7tB6SAURAWwALCAlhAWAALSi/ea71Wr5PhN3DmzP5hYiwQ2j9IkB7kOErDmlmM2az6x91VlZ4C0rPQKBTZGvOHpuejP07TeDpK7znWJ+0Nrt8O1LrtZtKDkJdMambffRgLBMcCodDqNulxzkB7WCnNVx2/e1DnUWVeW7uHE7Or44HQWCC0R99feSeh9K/CY8YnwuMT4Rt1pb9/8IXXSfu6d1aEI9tjgeCLlyuB2fS3FynHRlgpz6eXXswsN191fNtXw2Ve8XhyZjE2FghNzUfm31N3+y8jLIV1dJ3xcdQfGp8Idza6h7oqK9w8XNqvbe5Pz0eeTy/99GrlpJUb5rBxjZVWfn5e6zXnE2Z/cny7tBF4vToWCE8vROOJJOlxsozhsD5pa3B921dT5mDj3GvmTez59NKoP/h+ZYv0LBjxEFZKW4NrpLfaR+W112E8MeoP/TAZnni9us7OZuJF8BNWStdXnsfdlynJazm2PfrP0PRC9NlU+COlT9PgwltYKR2NLlKn9h81Gv+rFf/s6rOJ8NvlzdwPQAk+w0rJ5an9zt7h5Fxk7OgYc2cPFl+5DisF66n9cmxnPBB6MbP8j1fHnwNTOcbOsRBk/dQ+nkj+698fxgKhUX/wPA++qhP/YaVc/NR+fWvf/3p1PBCaWYxBT2dSS1jIp/bvlzcn5iKj/uD0fPQgztsxJj78X2Oldfqp/f5hYmYhOjUf/bs/+HZJvT/ZXYRKw/rk81P7eDw5NhF6Nhn2v15dI/qcPgfUHlbK7XrHxvbBzJsorS8FYo+KrrFO8cNkmPQIvIGndAAWEBbAAsICWEBYAAsIC2ABYQEsICyABYQFsICwABYQFsACwgJYQFgACwgLYAFhASwgLIAFhAWwgLAAFhAWwALCAlhAWAALCAtgAWEBLCAsgMV/AgAA//956aWpNSdLJgAAAABJRU5ErkJggg==
```
Comment (dgn) [inline manifests-maistra/1.0.0/maistraoperator.v1.0.0.clusterserviceversion.yaml]: this was probably not meant to be in this commit, right? looks like the upstream Istio icon

File: manifests-servicemesh/1.0.0/servicemeshoperator.v1.0.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -124,6 +123,9 @@ spec:
...
...gytbM4uxyNru5FxkY/uAxIAo1jb3/ZuraX/JbBDLnIqz2JRqzutUDLLWatEVWfSn/57/11MNYWUgtdW+s3cYXNl6t/zfjKJruxNzke3dQ9KjYbG5czg1F52ai/76l/Sy1utU3DaTJArOo084xSApJtluNWg0mlqftdxlWQiuk5j6NHl9331PeoY06iuL596t7e7HSQ9CNUkUykrNh/HkYniD9CzHUfqJNTkXIT0CAw4OE7Pv1khPkR7tB6SAURAWwALCAlhAWAALSi/ea71Wr5PhN3DmzP5hYiwQ2j9IkB7kOErDmlmM2az6x91VlZ4C0rPQKBTZGvOHpuejP07TeDpK7znWJ+0Nrt8O1LrtZtKDkJdMambffRgLBMcCodDqNulxzkB7WCnNVx2/e1DnUWVeW7uHE7Or44HQWCC0R99feSeh9K/CY8YnwuMT4Rt1pb9/8IXXSfu6d1aEI9tjgeCLlyuB2fS3FynHRlgpz6eXXswsN191fNtXw2Ve8XhyZjE2FghNzUfm31N3+y8jLIV1dJ3xcdQfGp8Idza6h7oqK9w8XNqvbe5Pz0eeTy/99GrlpJUb5rBxjZVWfn5e6zXnE2Z/cny7tBF4vToWCE8vROOJJOlxsozhsD5pa3B921dT5mDj3GvmTez59NKoP/h+ZYv0LBjxEFZKW4NrpLfaR+W112E8MeoP/TAZnni9us7OZuJF8BNWStdXnsfdlynJazm2PfrP0PRC9NlU+COlT9PgwltYKR2NLlKn9h81Gv+rFf/s6rOJ8NvlzdwPQAk+w0rJ5an9zt7h5Fxk7OgYc2cPFl+5DisF66n9cmxnPBB6MbP8j1fHnwNTOcbOsRBk/dQ+nkj+698fxgKhUX/wPA++qhP/YaVc/NR+fWvf/3p1PBCaWYxBT2dSS1jIp/bvlzcn5iKj/uD0fPQgztsxJj78X2Oldfqp/f5hYmYhOjUf/bs/+HZJvT/ZXYRKw/rk81P7eDw5NhF6Nhn2v15dI/qcPgfUHlbK7XrHxvbBzJsorS8FYo+KrrFO8cNkmPQIvIGndAAWEBbAAsICWEBYAAsIC2ABYQEsICyABYQFsICwABYQFsACwgJYQFgACwgLYAFhASwgLIAFhAWwgLAAFhAWwALCAlhAWAALCAtgAWEBLCAsgMV/AgAA//956aWpNSdLJgAAAABJRU5ErkJggg==
```
Comment (dgn) [inline manifests-servicemesh/1.0.0/servicemeshoperator.v1.0.0.clusterserviceversion.yaml]: same here

**Comment by rcernich** [comment]
Is there any harm to leaving the ior related helm templates?

File: manifests-maistra/1.0.0/maistraoperator.v1.0.0.clusterserviceversion.yaml (hunk 1 above)
Comment (dmitri-d) [reply manifests-maistra/1.0.0/maistraoperator.v1.0.0.clusterserviceversion.yaml]: Indeed not!

**Comment by dmitri-d** [comment]
> Is there any harm to leaving the ior related helm templates?

It seems that support for ior is done through templates only. I figured removing templates would make it easier to re-add them later (by reverting the commit).

**Review by rcernich — APPROVED** [review]

**Review by brian-avery — APPROVED** [review]

**Pull request merged**